cmd/seccomp.go

```diff
@@ -0,0 +1,137 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	apiv1 "k8s.io/api/core/v1"
+	k8sRuntime "k8s.io/apimachinery/pkg/runtime"
+)
+
+func checkSeccomp(resource k8sRuntime.Object, result *Result) {
+	annotations := getPodAnnotations(resource)
+	podAnnotation := apiv1.SeccompPodAnnotationKey
+	podVal, podOk := annotations[podAnnotation]
+
+	if podOk {
```

Comment:
If `!podOk` it will fall through and check all the containers. If all the containers have annotations then that is ok. If we want to enforce always having a pod annotation I can add that in.

Reply:
Ah just saw your comment below